FreeRTOS/include/FreeRTOSConfig.h

```diff
@@ -0,0 +1,129 @@
+/*
+ * FreeRTOS V202212.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
Please fix license headers. Can't be this, must be ours. If can't be fixed then tell user how to get these or move to examples repo.

Reply:
will do.  added those last-minute last night to get the bundle to build.